src/main/webapp/js/constants/field-configs.js

```diff
@@ -521,3 +523,62 @@ export const campaignMessageFieldConfig = [{
   format: 'yyyy-MM-dd HH:mm:ss',
 },
 ];
+
+export const appSettingsFieldConfig = [{
```

Comment:
I think you can move it to the app settings component, also you can remove the id, create and update date fields form the config